tox.ini

```diff
@@ -41,6 +41,8 @@ whitelist_externals = make
 
 [testenv:py26-django14]
 basepython = python2.6
+install_command = pip install --no-binary=Django {opts} {packages}
+recreate = True
```

Comment:
Explained in the commit message https://github.com/moreati/django-two-factor-auth/commit/b7d63814876f9d88aa201c59552ab6098599419c

> `recreate=True` makes sure any existing, wheel based, environments are
deleted. Once that's done, these lines could be removed.